back/qc_api/migrations/0004_snippet_type.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.1.2 on 2020-11-06 08:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I think we should add this folder in .gitignore

Reply:
#51